tests/console/mysql_odbc.pm

```diff
@@ -38,8 +47,8 @@ sub setup {
     # write odbcinst.ini
     assert_script_run "echo [myodbc_mysql] > /etc/unixODBC/odbcinst.ini";
     assert_script_run "echo Description=ODBC for MySQL >> /etc/unixODBC/odbcinst.ini";
-    assert_script_run "echo Driver=/usr/lib64/libmyodbc5.so >> /etc/unixODBC/odbcinst.ini";
-    assert_script_run "echo Setup=/usr/lib64/unixODBC/libodbcmyS.so >> /etc/unixODBC/odbcinst.ini";
+    assert_script_run "echo Driver=/usr/$arch_dir/libmyodbc5.so >> /etc/unixODBC/odbcinst.ini";
```

Comment:
I think better will be:
`assert_script_run "echo Driver=$(rpm --eval '%_libdir')/libmyodbc5.so >> /etc/unixODBC/odbcinst.ini";`

and also more portable on rpm based systems :)